src/components/common-components/modal/save-changes-modal/save-changes-modal.component.tsx

```diff
@@ -72,7 +72,7 @@ export interface SaveChangesModalOptions {
   setOpen: (open: boolean) => void;
   open: boolean;
   handleSave: () => void;
-  closeOptions: () => void;
+  closeOptions: (editMode: boolean) => void;
```

Comment:
@Ehevi 
Pamiętasz dlaczego nazwałaś tę funkcję `closeOptions`?
Czy nie powinno było być po prostu `closeModal`?

Reply:
W moim założeniu miała być odpowiedzialna za robienie tych rzeczy, które trzeba wykonać przed zamknięciem modalu, np. w przypadku report-issue-modal czyszczenie pola ze wpisanej wiadomości (i we wcześniejszej wersji usuwanie starego screenshota)

![report-issue-modal](https://user-images.githubusercontent.com/48785655/111863889-5de2ae00-895e-11eb-9140-a4ee148d7e7d.png)
![return](https://user-images.githubusercontent.com/48785655/111863892-5fac7180-895e-11eb-8c6d-7b88a46e572d.png)

A dla zapisywania zmian wychodzenie z trybu edycji (bo zmiany zapisujemy po kliknięciu na `Wyjdź`, więc tak by pasowało żeby wyjść 😛).

![edi-page-toolbar-options](https://user-images.githubusercontent.com/48785655/111864039-fc6f0f00-895e-11eb-9dae-9b80a97799b4.png)
![edit-page-savechangesmodal](https://user-images.githubusercontent.com/48785655/111864040-fda03c00-895e-11eb-97e3-6ec91f54cfd7.png)

Zgadzam się, słaba nazwa, lepiej byłoby np. `beforeClose` / `onModalClose` albo coś w tym rodzaju.